docs/src/code-examples/setup-configure/go.mdx

```diff
@@ -0,0 +1,9 @@
+---
+lang: go
+---
+
+```go
+import "github.com/govau/notify-client-go"
```

Comment:
I think that `goimports` will actually rewrite the import like this:

```go
import (
  notify "github.com/govau/notify-client-go"
)

client, err := notify.NewClient(apiKey)
```

So that might be a better example to use. Feel free to check out what `goimports` does.

Reply:
@dalehurwitz but this is minor.

Reply:
In my IDE `goimports` does not add the brackets for the single import

Reply:
@dalehurwitz yeah it's all good, non-issue.